doc/api/buffer.md

```diff
@@ -510,6 +510,11 @@ console.log(buf2.toString());
 ### Class Method: Buffer.alloc(size[, fill[, encoding]])
 <!-- YAML
 added: v5.10.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/17428
+    description: Specifying an invalid string for `fill` now yields a
```

Comment:
Just a thought but given `yield` is a JS keyword and not necessarily a super familiar word for non-native speakers, is there another word we could use? Maybe `results in`, `creates` or `returns`?

Reply:
You can say the same about `returns` ;) I’m happy with anything, `results in` sounds fine to me.